Source/ACE.Server/WorldObjects/Corpse.cs

```diff
@@ -397,6 +397,7 @@ public void TryGenerateRare(DamageHistoryInfo killer)
         /// </summary>
         public static HashSet<ushort> NoDrop_Landblocks = new HashSet<ushort>()
         {
+            0x00EA,     // Mhoire Armory
```

Comment:
should be after colo

all of these landblocks are specified in order

Reply:
Fixed